test/hotspot/jtreg/compiler/lib/template_framework/Renderer.java

```diff
@@ -85,7 +86,7 @@ static boolean isValidHashtagOrDollarName(String name) {
      *
      * <p>
      * Instead, the user should create a {@link TemplateToken} from the inner {@link Template}, and
```

Comment:
Read this through again to refresh my memory an noticed this: "Should" sounds like good advise but below it clearly mention it's forbidden. Would it make sense to swap the paragraphs? Then it makes it more clear that this is not only advise but the only way to go.